docs/community-resources.md

```diff
@@ -0,0 +1,8 @@
+## Community Resources
+
+### Talks
+
+- [EuroPython 2022](https://www.youtube.com/watch?v=AutugvJNVkY&)
+- [GeoPython 2022](https://www.youtube.com/watch?v=YCpbCQwbkd4)
+- [PyCon US 2022](https://www.youtube.com/watch?v=1IiL31tUEVk)
```

Comment:
https://youtu.be/1IiL31tUEVk?t=2101 - add a specific time from when the talk starts

Reply:
Thanks @shivaylamba . Implemented 😄